README.md

```diff
@@ -6,6 +6,10 @@
 [![image](https://img.shields.io/pypi/pyversions/ruff-lsp.svg)](https://pypi.python.org/pypi/ruff-lsp)
 [![Actions status](https://github.com/astral-sh/ruff-lsp/workflows/CI/badge.svg)](https://github.com/astral-sh/ruff-lsp/actions)
 
+**As of Ruff v0.4.5, Ruff includes a built-in language server written in Rust: `ruff server`.
+`ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no
+installation required. `ruff server` is currently in Beta. We'd love your feedback.**
```

Comment:
This would benefit from a link to the migration guide.

Reply:
We could put this as a Note to better highlight this. It would look like:

> [!NOTE]
>
> **As of Ruff v0.4.5, Ruff ships with a built-in language server written in Rust: ⚡ `ruff server` ⚡**
>
> **`ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no
> installation required. `ruff server` is currently in Beta. We'd love your feedback.**

```
> [!NOTE]
>
> **As of Ruff v0.4.5, Ruff ships with a built-in language server written in Rust: ⚡ `ruff server` ⚡**
>
> **`ruff server` supports the same feature set as `ruff-lsp`, but with superior performance and no
> installation required. `ruff server` is currently in Beta. We'd love your feedback.**
```